src/store/common/getters.js

```diff
@@ -68,3 +68,7 @@ export function defaultActiveAssetType (state, getters, rootState, rootGetters)
 
   return activeAssetTypes.find(assetType => assetType.isDefault)
 }
+
+export function stripeActive () {
+  return !!(process.env.VUE_APP_STRIPE_ACTIVE === 'true' && process.env.VUE_APP_STRIPE_PUBLISHABLE_KEY)
```

Comment:
Any specific reason for `process.env.VUE_APP_STRIPE_ACTIVE`. Just removing the publishable key may be enough to disable stripe.

Reply:
Indeed